core/src/main/java/nl/nn/adapterframework/pipes/Base64Pipe.java

```diff
@@ -85,10 +86,13 @@ public void configure() throws ConfigurationException {
 				setLineSeparatorArray(getLineSeparator());
 			}
 		}
-
-		// Allow this for backwards compatibility
-		if(!convertToString && getDirection().equals("decode")) {
-			setOutputType("bytes");
+		
+		if (dir==Direction.DECODE && convertToString) {
+			setOutputType("string");
+		}
+		
+		if (getOutputTypeEnum()==null) {
+			setOutputType(dir==Direction.DECODE ? "bytes" : "string");
```

Comment:
Als jij convertToString weghaalt vind ik dat je ook de default outputType op stream moet zetten.
Het is immers de lcm release

Reply:
zo beter?